lib/appmap.rb

```diff
@@ -112,17 +112,49 @@ def detect_metadata
   end
 end
 
+lambda do
+  Initializer = Struct.new(:class_name, :module_name, :gem_module_name)
+
+  INITIALIZERS = {
+    'Rails::Railtie' => Initializer.new('AppMap::Railtie', 'appmap/railtie', 'railtie'),
+    'RSpec' => Initializer.new('AppMap::RSpec', 'appmap/rspec', 'rspec-core'),
+    'Minitest::Unit::TestCase' => Initializer.new('AppMap::RSpec', 'appmap/minitest', 'minitest')
+  }
+
+  TracePoint.new(:class) do |tp|
+    cls_name = tp.self.name
+    initializers = INITIALIZERS.delete(cls_name)
+    if initializers
+      initializers = [ initializers ] unless initializers.is_a?(Array)
```

Comment:
```suggestion
      initializers = Array(initializers)
```

Also, none of them is an array currently, so maybe there's no need to handle multiple?

Reply:
In another branch, there is a multiple.